src/client/components/heros/Faq.hero.vue

```diff
@@ -2,7 +2,7 @@
 
     <div id="faqSection">
 
-        <span class="alignCenter bigMarginTop pageTitle">FAQ</span>
+        <span class="alignCenter bigMarginTop pageTitle">Cryptocurrency explained</span>
```

Comment:
We should let it FAQ because we're not explaining cryptocurrency on this page

Reply:
Explain about cryptocurrency WebDollar, it is very good, optimal. And do not change the name of the section in the main menu. It's all right, believe me.

Reply:
I think FAQ is better suited in this scenario as well